### PR TITLE
vector_store.create_index is now create_collection, not recreate

### PR DESCRIPTION
**Pull request by drwsctt**
## Purpose of change

vector_store.create_index will do qdrant_client.create_collection instead of recreate_collection.
This will allow the end user to do the following and avoid a nasty bug in the current version of Qdrant:
  vector_store.delete_index("fubar")
  time.sleep(10)
  vector_store.create_index("fubar", 768)

## Description of change

create_index -> create_collection (changed)
recreate_index -> recreate_collection (added)

## Tests of change

Modified in place in my site-packages directory and ran variations of create/delete/recreate
Note that creating an index that already exists results in error, just as it would for qdrant_client

### CODE REVIEW

File: seaplane-sdk/python/src/seaplane/vector/vector_store.py [hunk 1]
```diff
@@ -65,6 +65,19 @@ def create_index(
...
+        return client.create_collection(
+            collection_name=name,
+            vectors_config=VectorParams(
+                size=vector_size, distance=self._map_to_qdrant_distance(distance)
+            ),
+            timeout=self.TIME_OUT_IN_SECONDS,
+        )
+    
+    def recreate_index(
+        self, name: str, vector_size: int = 1024, distance: SPDistance = SPDistance.COSINE
```
Comment (fokkedekker) [inline seaplane-sdk/python/src/seaplane/vector/vector_store.py:77]: @drwsctt should we set the default to 762 for Seaplane embeddings?

**Review by fokkedekker — APPROVED** [review]
LGTM!

File: seaplane-sdk/python/src/seaplane/vector/vector_store.py (hunk 1 above)
Comment (drwsctt) [reply seaplane-sdk/python/src/seaplane/vector/vector_store.py:77]: Yeah, let me do that (768)

**Comment by drwsctt** [comment]
Or not... I don't know if I have a different version of `black`, or what. It failed to lint, I made changes according to `black --check` and it still fails. :(

**Comment by tonilopezmr** [comment]
Maybe you aren't using `black` from `poetry`, you have to do `poetry shell` and then `black .` or run `poetry run black .`

**Comment by drwsctt** [comment]
That was it. Thanks!

**Pull request merged**